pombola/south_africa/templates/south_africa/mp_attendance.html

```diff
@@ -91,16 +91,21 @@ <h2 class="average-attendance">{{ aggregate_attendance }}%</h2>
       <th>Name</th>
       <th>Party</th>
       <th>Meetings</th>
+      {% if position = 'mps' %}
```

Comment:
I think this was changed later, to what you are asking here in the second point. Could you perhaps just check and see if it's okay?